cmake/windows/resources/installer-Windows.iss.in

```diff
@@ -7,7 +7,7 @@
 ; NOTE: The value of AppId uniquely identifies this application.
 ; Do not use the same AppId value in installers for other applications.
 ; (To generate a new GUID, click Tools | Generate GUID inside the IDE.)
-AppId={{CD703FE5-1F2C-4837-BD3D-DD840D83C3E3}
+AppId={{@UUID_APP@}
```

Comment:
@PatTheMav is either the original or updated (and still current) `{{` a typo, or is the double open curly brace correct?